xml/libvirt_guest_installation.xml

```diff
@@ -89,14 +85,12 @@
       </listitem>
      </varlistentry>
      <varlistentry>
-      <term><guimenu>Network installation (HTTP, FTP, or NFS)</guimenu>
+      <term><guimenu>Network install (HTTP, HTTPS, or FTP)</guimenu>
```

Comment:
I preferred "installation", to be honest.

CD-ROM or DVD-ROM? Isn't DVD more likely?

Reply:
Me too but that's what the GUI says now...